progenitor/tests/build_nexus.rs

```diff
@@ -31,6 +31,11 @@ mod builder_untagged {
             spec = "../sample_openapi/nexus.json",
             interface = Builder,
             tags = Merged,
+            adjustments = {
+                "Name" = {
+                    derives = [Hash],
+                }
+            }
```

Comment:
@andrewjstone this is the syntax I'm thinking about, but would welcome ... something better. I don't want to just do

```rust
per_type_derives = {
    "Name" = [Hash]
}
```

... because I think it makes more sense to lump together various tweaks one might make to a type... although I can't readily think of more...